packages/aws-cdk-lib/custom-resources/test/aws-custom-resource/aws-custom-resource.test.ts

```diff
@@ -46,13 +46,23 @@ test('aws sdk js custom resource with onCreate and onDelete', () => {
       'physicalResourceId': {
         'id': 'loggroup',
       },
+      'logHandlerEvent': true,
+      'logApiResponse': true,
+      'logResponseObject': true,
+      'logSdkVersion': true,
+      'logErrors': true,
```

Comment:
Calling out here that while this is an existing unit test being updated, it is not a breaking change. `AwsSdkCall` now has logging configured as `on` by default. Thus, the event passed to the Lambda has additional properties. This maintains the current default behavior in the handler code by logging all data by default.